rplugin/python3/denite/scantree.py

```diff
@@ -0,0 +1,48 @@
+# ============================================================================
+# FILE: scantree.py
+# AUTHOR: Jorge Rodrigues <skeept at gmail.com>
+# License: MIT license
+# ============================================================================
+
+import sys
+from os.path import basename, join
+from os import walk
+
+try:
+    from os import scandir
+
+    def scantree(path_name, skip_list=None):
+        """if scandir is available use it as it is faster than os.walk"""
```

Comment:
Well, this is kind a personal reason but the document comment should explain what the function stands for. This kind of comment should be in after the summary line or a normal command.